packages/hebao_v1/contracts/stores/SecurityStore.sol

```diff
@@ -53,83 +55,165 @@ contract SecurityStore is DataStore
         view
         returns (bool)
     {
-        return wallets[wallet].guardianIdx[guardian] > 0;
+        return wallets[wallet].guardianIdx[guardian] > 0 &&
+            isGuardianValid(wallet, guardian);
     }
 
+    function getGuardianValidTs(
+        address wallet,
+        address guardian
+        )
+        public
+        view
+        returns (uint, uint)
+    {
+        return (wallets[wallet].guardianValidSince[guardian],
+                wallets[wallet].guardianValidUntil[guardian]);
+    }
+
+
```

Comment:
If we want to expose the timestamps anyway, what about just adding the `validSince`and `validUntil` to `Data.Guardian`? This way all data for the guardian is nicely together is also directly available in the other getters we already have for the guardians.

Reply:
Yes, I agree. updated.